lua/tardis/parts/legacy/legacy_unused.lua

```diff
@@ -0,0 +1,21 @@
+--Legacy Interior - Unused Pull Handle
+
+local PART = {}
+PART.ID = "legacy_unused"
```

Comment:
Should this perhaps be renamed the advanced handle since it has a use?

Reply:
Doesn't have a use anymore. Was mostly used for testing